### ISSUE TITLE
Idea: per-user outbound feed

### ISSUE THREAD

**Issue by kfogel** [feedbin/feedbin#180]
When I'm reading on Feedbin, I often want to share some of the posts I read with others -- ideally, I'd like to do so via a feed.  This would be a feed that anyone (not only other feedbin users) could subscribe to via some obvious link like https://feedbin.com/kfogel/rss.

From my side, as the "publisher", the UI would be simple: another icon, up on the top toolbar of per-post icons.  Here's a sample screenshot with the hypothetical new sharing icon added _(update 2016-06-23: see my [later comment](https://github.com/feedbin/feedbin/issues/180#issuecomment-228091932) -- I didn't even know about the existing sharing option until @DavidOliver pointed it out, and now I think this idea could be more easily implemented as just another option in the sharing services menu)_:

![feedbin-per-user-outbound-rss-example](https://cloud.githubusercontent.com/assets/401111/16282658/45049a94-388f-11e6-81c1-8bd9615a7e0e.png)

(It's the little open triangle right above the "16" in "2016".  I somewhat randomly chose the generic "share" icon from the Google Material Design icon set, but maybe it should be an RSS symbol instead or something.)

When the user clicks on the icon, a dialog box would open up, asking "Publish this post on your feed, so others know you recommend it?" or something like that.  (I do think a confirmation step is necessary, since one's reading habits are private by default; confirmation is okay as long as it's not habituatable -- and here it would not be habituatable, since the _other_ things one might have meant to click instead would not normally cause that dialogue to pop up.)

I guess this is a cheap way to turn Feedbin into a social network, or at least into a piece of a social network :-).  But even leaving out grand and quixotic visions of a decentralized web, cough cough, it would still be a great feature for enabling Feedbin users to recommend stuff (and, not coincidentally, enable them to advertise Feedbin along the way).

By the way, to see if anyone else was interested in this, I [tweeted](https://twitter.com/kfogel/status/745721190463967232) and [dented](https://identi.ca/kfogel/note/MSwhURUPQAKTFYXTllNZrA) about it.

**Comment by DavidOliver** [comment]
While a per-user feed could be useful I think it would be very limiting compared to being able to add a post to one of potentially multiple Feedbin-generated feeds. I would like to be able to create one or many feeds, with each being for a different purpose. E.g., `techie-friends`, `game-friends` and `publish-on-website`.

I currently "share" with [Pinboard](https://pinboard.in/) from Feedbin with tags to achieve this, as Feedbin provides a feed for each tag. But it would be great to have such a feature available as part of Feedbin itself.

**Comment by kfogel** [comment]
Hmm, interesting idea.  There's a choices-vs-simplicity tradeoff here, UI-wise, I think, because Feedbin would have to have an interface for managing the different outbound feeds for a given user.  Personally, I'd want the "choices" side too, but it might make sense to implement the simple thing first, since adding the multiple-different-feeds feature can always be done on top of that, if people want it, with no backtracking.

(Probably people's mental model of how "share with friends" works is greatly affected by however Facebook works these days!  I'm not on Facebook, so don't know the UI or conceptual model there.)

**Comment by DavidOliver** [comment]
Yeah, true. That's why I mentioned some of the other forms of sharing Feedbin already has such as email, which do have options in the slide-down panel, so options within a sharing method should be a familiar concept to Feedbin users.

What I have in mind is:

1. Press share button (which doesn't appear in your screenshot - I guess you don't have any sharing options set up)
2. Select 'Feed' or whatever it were called
3. Apply tag(s), with each tag correlating to a feed name.

**Comment by kfogel** [comment]
I never even knew there were sharing options, until you mentioned it.  I never had a reason to open that menu before.  Now I see it, and even see that there is a "Custom Services" option, meaning I could script this up myself if I wanted to.

I still think a per-user outbound feed would be a great feature, but maybe the way to do it is just to offer it as one of the many sharing services listed by default -- just this service happens to be provided by Feedbin.
